host/sgx/sgx_enclave_common_wrapper.c

```diff
@@ -168,6 +185,40 @@ void* oe_sgx_enclave_create(
         enclave_error);
 }
 
+void* oe_sgx_enclave_create_ex(
+    void* base_address,
+    size_t virtual_size,
+    size_t initial_commit,
+    uint32_t type,
+    const void* info,
+    size_t info_size,
+    const uint32_t ex_features,
+    const void* ex_features_p[32],
+    uint32_t* enclave_error)
+{
+    _load_sgx_enclave_common();
+    if (ex_features)
+        return _enclave_create_ex(
```

Comment:
_enclave_create_ex can be null if zero-based enclaves are requested with old PSW. In such case, detect it, print a fatal message and abort.

Reply:
done.

ratrames@accvm:~/dev/OE_0_base/build/tests/zerobase$ ctest -V
UpdateCTestConfiguration  from :/home/ratrames/dev/OE_0_base/build/tests/zerobase/DartConfiguration.tcl
UpdateCTestConfiguration  from :/home/ratrames/dev/OE_0_base/build/tests/zerobase/DartConfiguration.tcl
Test project /home/ratrames/dev/OE_0_base/build/tests/zerobase
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 1
    Start 1: tests/zerobase

1: Test command: /home/ratrames/dev/OE_0_base/build/tests/zerobase/host/zerobase_host "/home/ratrames/dev/OE_0_base/build/tests/zerobase/enc/zerobase_enc"
1: Test timeout computed to be: 10000000
1: Could not create enclave, result=21
1: 2021-06-30T02:07:40+0000.238811Z [(H)ERROR] tid(0x7fdbcb234740) | enclave_create_ex function not found.
1:  [/home/ratrames/dev/OE_0_base/host/sgx/sgx_enclave_common_wrapper.c:_lookup_function:110]
1: 2021-06-30T02:07:40+0000.238819Z [(H)ERROR] tid(0x7fdbcb234740) | enclave_create_ex() was not found in installed libsgx_enclave_common.so.1. [/home/ratrames/dev/OE_0_base/host/sgx/sgx_enclave_common_wrapper.c:oe_sgx_enclave_create_ex:202]
1: 2021-06-30T02:07:40+0000.238821Z [(H)ERROR] tid(0x7fdbcb234740) | enclave_create with ENCLAVE_TYPE_SGX1 type failed (err=0) (oe_result_t=OE_PLATFORM_ERROR) [/home/ratrames/dev/OE_0_base/host/sgx/sgxload.c:oe_sgx_create_enclave:441]
1: 2021-06-30T02:07:40+0000.238822Z [(H)ERROR] tid(0x7fdbcb234740) | :OE_PLATFORM_ERROR [/home/ratrames/dev/OE_0_base/host/sgx/create.c:oe_sgx_build_enclave:964]
1: 2021-06-30T02:07:40+0000.238962Z [(H)ERROR] tid(0x7fdbcb234740) | :OE_PLATFORM_ERROR [/home/ratrames/dev/OE_0_base/host/sgx/create.c:oe_create_enclave:1204]
1/1 Test #1: tests/zerobase ...................***Failed    0.00 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.00 sec

The following tests FAILED:
          1 - tests/zerobase (Failed)
Errors while running CTest